views/shared/profiles/stats.jade

```diff
@@ -4,6 +4,10 @@ table.table.table-striped
     td
       strong=env.t('health')
       | : {{profile.stats.hp | number:0}} / 50
+  tr(ng-if='profile.stats.lvl >= 10 && !profile.preferences.disableClasses')
```

Comment:
The header uses: `user.flags.classSelected && !user.preferences.disableClasses`, but flags aren't accessible through the member service, so I chose the next best thing.